sig-usability/charter.md

```diff
@@ -0,0 +1,54 @@
+# SIG Usability Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers
+topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes
+project is maximaly usable by users of a variety of different backgrounds and abilities. Examples
+of efforts include internationalization and accessibility of documentation.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- usability of end-user facing experiences (error messages, end-to-end tasks, etc)
+- accessibility of Kubernetes community artifacts, examples include:
+   - internationalization of documentation in binaries and documentation
+   - color choices for people with color blind-ness
+   - ensuring compatibility with screen reader technology
+- user interface design for core components with user interfaces.
+- translations - https://github.com/kubernetes/kubernetes/tree/master/translations
+- internationalization - https://github.com/kubernetes/kubernetes/tree/master/pkg/kubectl/util/i18n
+
+#### Cross-cutting and Externally Facing Processes
+
+- None currently, eventually perhaps a more formal usability review for changes
+
+### Out of scope
+
+* Contributor user experience (covered by SIG-Contributor Experience)
+* API Design (covered by SIG-Architecture)
+* Command line tool infrastructure (covered by SIG-CLI)
+
+## Roles and Organization Management
+
+This sig follows adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+### Additional responsibilities of Chairs
+None
+
+### Deviations from [sig-governance]
+
+No Tech Leads (for now)
```

Comment:
It would be great to have Tech Leads as part of SIG Usability that can collaborate with ux researchers to make sure that ongoing research with end users has the depth it needs to be useful to the community.

Reply:
Happy to add people as part of growing the SIG, but I think that we need to grow tech. leads organically through the operation of the SIG rather than identify them as part of formation.